packages/flutter/lib/src/material/theme_data.dart

```diff
@@ -200,8 +200,17 @@ enum MaterialTapTargetSize {
 class ThemeData with Diagnosticable {
   /// Create a [ThemeData] that's used to configure a [Theme].
   ///
-  /// Typically, only the [brightness], [primaryColor], or [primarySwatch] are
-  /// specified. That pair of values are used to construct the [colorScheme].
+  /// Typically, only the [brightness] and [colorSchemeSeed] are specified.
+  /// This pair of values is used to construct the [colorScheme] by using
+  /// [ColorScheme.fromSeed].
+  ///
+  /// For historical reasons, instead of using a [colorSchemeSeed], you can
```

Comment:
Good to note this, also might be worth noting the primary alternative is to use `ColorScheme.fromSeed`,  customize as needed with `copyWith` and set `colorScheme`

Reply:
Good point. I updated the comment with something about that.